cdap-app-fabric/src/test/java/co/cask/cdap/metadata/SystemMetadataAuditPublishTest.java

```diff
@@ -92,13 +90,13 @@ public void testPublishing() throws Exception {
     AppFabricTestHelper.deployApplication(Id.Namespace.DEFAULT, AllProgramsApp.class, null, cConf);
     Set<String> addedMetadata = new HashSet<>();
     // TODO (CDAP-14670): this test is brittle, find a better condition to wait on
-    Tasks.waitFor(25, () -> addAllSystemMetadata(addedMetadata), 10, TimeUnit.SECONDS);
+    Tasks.waitFor(26, () -> addAllSystemMetadata(addedMetadata), 10, TimeUnit.SECONDS);
     namespaceAdmin.delete(NamespaceId.DEFAULT);
     Set<String> removedMetadata = new HashSet<>();
-    // TODO (CDAP-14666): deletions have one additional key: "profile". This is because of named bug.
```

Comment:
As mentioned above, CDAP-14666 is now fixed and we see the same number of messages for deploy as we see for delete.